cli/azd/pkg/auth/manager_test.go

```diff
@@ -61,6 +64,7 @@ func TestServicePrincipalLoginClientSecret(t *testing.T) {
 		configManager:     newMemoryConfigManager(),
 		userConfigManager: newMemoryUserConfigManager(),
 		credentialCache:   credentialCache,
+		cloud:             publicCloud,
```

Comment:
I'd like to really avoid having to support `cloud` as a top level member of our components and move to a model where it is an internal implementation detail of working with the Go SDK.